src/Maker/Security/MakeFormLogin.php

```diff
@@ -57,11 +59,11 @@ final class MakeFormLogin extends AbstractMaker
 
     private const SECURITY_CONFIG_PATH = 'config/packages/security.yaml';
     private YamlSourceManipulator $ysm;
-    private string $controllerName;
     private string $firewallToUpdate;
     private string $userClass;
     private string $userNameField;
-    private bool $willLogout;
```

Comment:
We need to keep both of these properties. MakerBundle is getting away from "abusing" `setArgument` and `getArgument` for the purposes of passing user input from one method to the next within a maker command. e.g. separation of concerns. 

tl;dr `interact()` configures and sets any values onto their respective properties that will later be needed in `generate()`

`interact()` should get, validate, and set input.
`generate()` should only use values that are set via class properties when possible.

Reply:
ok so we don't have to use --no-interaction option for all makers ?